src/app/app.component.html

```diff
@@ -22,7 +22,19 @@
     </mat-form-field>
   </div>
 
-  <div class="dataDiv">
-    <app-weather-data *ngIf=" locationID || locationID == 0 " [data]="weatherData[locationID]"></app-weather-data>
+  <div class="centerPage">
+    <ng-container *ngIf=" locationID || locationID == 0">
+      <div class="timeBtns">
+        <b>Select a Time</b>
+        <ng-container *ngFor=" let timeStep of weatherData[locationID].timeSeries; index as i ">
+          <ng-container *ngIf="(weatherData[locationID].timeSeries[0].time | date:'d') == (timeStep.time | date:'d')">
+            <button *ngIf="i<=24" (click)=" timeID = i ">{{timeStep.time | date:'shortTime'}}</button>
```

Comment:
very minor, but you've got some unnecessary spaces in a couple of these (ngFor and click)

Reply:
I believe i might have sorted then now? :p i think the ctrl + alt + L did this :p

Reply:
comment was more about spaces after and before `"` rather than on the `=`:
`(click)="timeID = i"` 

looks like those two lines still have a trailing space before the closing `"`